cmd/connector/validate/entitlement_read.go

```diff
@@ -42,14 +43,17 @@ var entitlementReadChecks = []Check{
 				res.warnf("no entitlements")
 				return
 			}
-
+			count := 0
```

Comment:
Could we randomize the set of accounts/entitlements that we are testing? I think running `rand.Shuffle` on `entitlements`/`accounts` before this should do it.

Reply:
Done